bot/cogs/snakes.py

```diff
@@ -222,7 +225,7 @@ def __init__(self, bot: AutoShardedBot):
         return snake_dict
 
     @command()
-    async def get(self, ctx: Context, name: str = None):
+    async def get(self, ctx: Context, name: to_lower = None):
```

Comment:
you can't typehint something to a function. `to_lower` is not a type, and as far as I can tell `name` is still a `str`

Reply:
You can do that, discord.py will convert the string to lowercase automatically